lib/iron_bank/user.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module IronBank
+  # A Zuora user, only available if the user data has been exported and provided
+  # to IronBank through the `zuora_users_file` configuration option.
+  #
+  # Cf. https://knowledgecenter.zuora.com/CF_Users_and_Administrators/A_Administrator_Settings/Manage_Users#Export_User_Data_to_a_CSV_File
+  #
+  class User
+    extend IronBank::Local
+
+    class << self
+      def store
+        zuora_users_file = IronBank.configuration.zuora_users_file
+        return {} unless zuora_users_file
+
+        @store ||= begin
+          if File.exist?(zuora_users_file)
+            load_records
+          else
```

Comment:
IMO, this feels too defensive; I'd let this fail if a `zuora_users_file` is specified but the file doesn't exist.

Reply:
As discussed, I'm in favor of keeping the behavior consistent with `excluded_fields` configuration, as seen in https://github.com/zendesk/iron_bank/blob/aa2bf04b20e8371a32c3539425903812d9f3b8bd/lib/iron_bank/configuration.rb#L88-L92.